backend/de.metas.swat/de.metas.swat.base/src/main/java/de/metas/picking/api/PackageableQuery.java

```diff
@@ -57,6 +59,8 @@ public class PackageableQuery
 	@Nullable WarehouseId warehouseId;
 	@NonNull @Singular ImmutableSet<LocalDate> deliveryDays;
 	@Nullable LocalDate preparationDate;
+	@Builder.Default @Nullable ZonedDateTime maximumPreparationDate = Env.getZonedDateTime();
+	@Builder.Default @Nullable ZonedDateTime maximumPromisedDate = Env.getZonedDateTime();
```

Comment:
hmm,
1. not sure about having these as defaults....
2. i also feel like the name is not intention revealing because if enforced, we are selecting those records without a fixed date or the fixed date <= maximum

Reply:
maybe call it maximumFixedPreparation, maximumFixedPromisedDate, something like this....